core/cli/test/index.test.ts

```diff
@@ -121,4 +122,24 @@ describe('cli', () => {
       throw e
     }
   })
+
+  it('should successfully install when options for different hooks are defined', async () => {
```

Comment:
**praise**: great test case!

Reply:
I don't _really_ love putting it in this file as none of the other tests are checking installs but i think it's good to get into the habit of adding tests when we find bugs